dev-requirements.txt

```diff
@@ -1,5 +1,4 @@
-percy
-selenium
 pandas
 xlrd
 flake8
+mimesis;python_version>="3.6"
```

Comment:
there are two major data faker: mimesis, faker

mimesis only support 3.6+, but much faster for large dummy data generation.